tests/test_wave_03.py

```diff
@@ -143,7 +139,6 @@ def test_mark_incomplete_missing_task(client):
     # Assert
     assert response.status_code == 404
 
-    # raise Exception("Complete test with assertion about response body")
-    # *****************************************************************
-    # **Complete test with assertion about response body***************
-    # *****************************************************************
+    # Added assertions
+    assert "'message': 'Task #1 was not found' in response_body"
```

Comment:
Same comment as above here -- this line is not comparing anything since it's the assert keyword and a string. For a test to test something there needs to be something compared to something else. 

This assertion could be written as
```python
assert response_body == {"message": "Task #1 was not found' in response_body"}